yang/ietf-schedule.yang

```diff
@@ -540,4 +570,60 @@ module ietf-schedule {
         "Defines a list of exceptions for recurrence.";
     }
   }
+  
+  grouping schedule-status {
+    description
+      "This grouping is defined to identify common properties of 
+       scheduling status.";
+    leaf schedule-id {
+      type string;
+      description
+        "The schedule identifier that globally identifies a schedule in
+          a device, controller, network, etc. The unicity scope depends
+          on the implementation.";
+    }
+    leaf schedule-type {
+      type identityref {
+        base schedule-type;
+      }
+      description
+        "The schedule type";
+    }    
+    leaf last-update {
+      type yang:date-and-time;
+      description
+        "The timestamp that the schedule is last updated.";
+    }
+    leaf version {
+      type uint16;
+      description
+        "The version number of the schedule.";
+    }
+    leaf state {
+      type identityref {
+        base schedule-state;
+      }
+      description
+        "The current state of the schedule.";
+    }
+    leaf counter {
+      when "derived-from-or-self(../schedule-type, 'schedule:recurrence')";
+      type uint32;
+      description
+        "The counter of occurrences since the schedule was enabled.";
```

Comment:
```suggestion
        "The counter of occurrences since the schedule was enabled.
          The count wraps around when it reaches the maximum value.";
```